.dockerignore

```diff
@@ -1,7 +1,7 @@
 .git
-test/**
-spec/**
 tmp/**
```

Comment:
Why did you remove those? Running `rails spec` locally will probably still create them?

Reply:
I removed them in order to include them in the Docker Image.

As mentioned in 2ba17a02a0640536d73eed3995a33aaf55df80f9 I would have liked to only include them in the test image, but that doesn't really work, there is also a issue here with the same problem https://github.com/moby/moby/issues/15771